content/en/apps/guides/offline/_index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Offline CHT Server"
+linkTitle: "Offline CHT"
+weight: 100
+description: >
+ Running an instance of CHT Core server with no Internet
+---
+
+## Introduction
+
+The CHT is built as an offline first application. This applies to clients, either  browsers or Android applications, connecting to the CHT server.  The server itself assumes it has Internet connectivity to provide services such as DNS, software updates and general use connectivity.  This document explores what it looks like when the CHT server is offline without these services available.
+
+Running a CHT server offline requires no modifications to the CHT itself.  Instead, supporting services normally found online are replicated locally. 
+
+## Considerations
+
+An offline CHT server is most appropriate for a development environment.  There are serious implications to consider before deploying an offline server in a production environment which are most often ignored for development.  Offline solution should have a plan in place for:
+
+* Alerting - How will alerts be sent in the case of downtime or degraded service? 
+* Power failures and unplanned restarts - Will the server cleanly restart such that the CHT resumes service correctly?
+* Backups - What happens to the CHT data if there's a hard drive failure?  
+* Disaster Recovery - What happens if there is a flood at the facility and active and backup data on site are destroyed?
+* Scale - What happens when the hardware deployed needs to be upgraded to increase capacity?
+* Updates - By definition TLS certificates expire and software needs to be updated - how will the deployment get these updates on a regular basis?
```

Comment:
Add something about privacy and the need to be physically secure and use disk encryption.

Reply:
Ah -  excellent point - thanks! I'll grab this tomorrow.

Reply:
nope, I grabbed it tonight in d1ef50b  ;)